.github/workflows/gpu-tests.yaml

```diff
@@ -28,6 +28,22 @@ jobs:
           retention-days: 5
       - run: echo "This job's status is ${{ job.status }}."
 
+  LULESH-tests:
+    runs-on: [self-hosted, linux, x64, build]
+    container: gcr.io/gem5-test/gcn-gpu:latest
+    needs: build-gem5
+    timeout-minutes: 600 # 10 hours
```

Comment:
@mkjost0 similar to other apps, how did you determine this timeout?  My recollection is that LULESH sometimes runs longer than this when running the default 10 iterations -- maybe @abmerop remembers differently though...

Reply:
A full run takes around 15 hours on my machine.  I usually exit after the first iteration when testing now though. I am not sure how many this test is trying to run

Reply:
Melissa is no longer working on the project. I don't know how she got the timeout number. How long would you recommend setting the timeout?

Reply:
24 hours should be sufficient... But I would like to discuss how redundant the run for this app is.  We don't gain much testing the default 10 iterations in this app vs. one, for example.  And with one it would take ~2 hours.